docs/general/dql/selects.rst

```diff
@@ -797,6 +825,58 @@ Object property can also be addressed in the :ref:`where clause
     +------------+----------------------------------------------------------------------+
     SELECT 1 row in set (... sec)
 
+For displaying the object's keys, use the :ref:`scalar-object_keys` function::
+
+    cr> select name, object_keys(inhabitants) as inhabitant_keys from locations
+    ... where name = 'Betelgeuse';
+    +------------+-------------------------+
+    | name       | inhabitant_keys         |
+    +------------+-------------------------+
+    | Betelgeuse | ["name", "description"] |
+    +------------+-------------------------+
+    SELECT 1 row in set (... sec)
+
+By combining the :ref:`scalar-object_keys` function with the ``ANY`` operator,
+you can find all records where an object contains a specific object key::
+
+    cr> select name from locations
+    ... where 'interests' = ANY(object_keys(inhabitants));
```

Comment:
Doing a `inhabitants['interests'] IS NOT NULL` is probably faster, using `ANY(<somescalar>)` cannot utilize the index.

Reply:
Thank you. Your suggestions are very reasonable.

> We could consider having some sort of "working with arrays and objects in CrateDB" tutorial or guide.

That's a good idea.